src/dns-test.js

```diff
@@ -293,19 +341,43 @@ async function sendQueries(nameservers_ipv4) {
  */
 function sendTelemetry(payload) {
     payload.measurementID = measurementID;
-    browser.telemetry.submitPing(TELEMETRY_TYPE, payload, TELEMETRY_OPTIONS);
+    console.log(payload);
+    // browser.telemetry.submitPing(TELEMETRY_TYPE, payload, TELEMETRY_OPTIONS);
```

Comment:
Remove `console.log(payload);` and uncomment the `submitPing` call.

Reply:
Addressed by 7a46895